packages/ts-moose-lib/src/dataModels/dataModelTypes.ts

```diff
@@ -12,6 +12,7 @@ export interface Column {
   required: boolean;
   unique: false; // what is this for?
   primary_key: boolean;
+  jwt: boolean;
```

Comment:
I think this field should go inside `Nested`?
Because this can only be in an object type.